Client/Frontend/AuthenticationManager/AuthenticationSettingsViewController.swift

```diff
@@ -221,6 +268,14 @@ class AuthenticationSettingsViewController: SettingsTableViewController {
             navigationItem.title = AuthenticationStrings.passcode
         }
     }
+    
+    private var authenticationSectionTitle: NSAttributedString {
+        if LAContext().canEvaluatePolicy(.DeviceOwnerAuthenticationWithBiometrics, error: nil) {
+            return NSAttributedString(string: AuthenticationStrings.authenticateFeaturesWithTouchIDPasscode)
+        } else {
+            return NSAttributedString(string: AuthenticationStrings.authenticateFeaturesWithPasscode)
+        }
```

Comment:
That would make sense to me. In that case, would we also want to change the actual settings menu item to something along the lines of `Authentication` instead of `Passcode` and `Touch ID & Passcode`?